schedules_tools/models.py

```diff
@@ -56,14 +61,20 @@ class Task(object):
     _schedule = None
     _subtree_hash = None
 
-    def __init__(self, schedule=None, level=1):
+    default_subtree_hash_attributes = ['name', 'dStart', 'dFinish']
+
+    def __init__(self, schedule=None, level=1, subtree_hash_extra=None):
```

Comment:
remove subtree_hash_extra - pass it dynamically to subtree_hash method from Diff class